acts-as-taggable-on.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = 'https://github.com/mbleigh/acts-as-taggable-on'
   gem.license       = 'MIT'
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files`.split($/).select { |f| f.match?(%r{\A((db|lib)/|LICENSE\.md)}) }
```

Comment:
I do not maintain this gem, but since this change is being proposed, any chance to switch to `Dir[...]` ?

Ref: https://docs.rubocop.org/rubocop-packaging/cops_packaging.html#packaginggemspecgit

Reply:
@patrickarnett  , thanks for the effort.
Let use Dir to have this PR merged.